scanomatic/data_processing/norm.py

```diff
@@ -89,8 +89,8 @@ def infer_offset(arr):
 
 
 def get_downsampled_plates(
-    data,
-    subsampling: Literal['TL', 'TR', 'BL', 'BR'] = "BR",
+    data: np.ndarray,
+    subsampling: Union[List[np.ndarray], Literal['TL', 'TR', 'BL', 'BR']] = "BR",
```

Comment:
in python 3.9 we don't need `List`, `Tuple`, `Dict` anymore, we can use `list`, `tuple` `dict` directly. Various places below

Reply:
ok, fixed.